docs/EDID to Receiver Caps Mapping.md

```diff
@@ -6,49 +6,34 @@
 
 {:toc}
 
-Extended Display Identification Data (EDID) is a metadata format for an apparatus to describe its capabilities as a video source (e.g. graphics card or set-top box). The data format is defined by a standard published by the Video Electronics Standards Association (VESA). This document is targeted against [E-EDID A2][E-EDID] which consist of EDID 1.4 (and covers EDID 1.3) and is referred to as _Base EDID_ and the [CTA-861-G][CTA-861] Extension Block imposed by HDMI. The information present in the EDID is subject to the requirements of the [Display Monitor Timing (DMT)][DMT] specification Version 1 Rev 12.
-
-[BCP-004-01][BCP-004-01] Receiver Capabilities defined a methodology for describing the Receivers with constraints on the properties of streams which related to [IS-04][IS-04] Senders. This is used to populate Receiver Capabilities for use with [IS-11][IS-11] Sink Metadata Processing.
-
-## Scope
-
-This document provides Best Current Practice guidelines for how to implement a mapping of EDID fields to Receiver Capabilities in order to have a specific way of representing an EDID Sink's Device through an associated [IS-04][IS-04] Receiver.
-
-## Use of Normative Language
-
-The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119][RFC-2119].
-
-## Normative References
-
-These appear at the end of the Markdown source for this document, and are referenced as hyperlinks within the main body.
+## Introduction
 
-## Definitions
+The purpose of AMWA BCP-005-01 is to provide Best Current Practice guidelines for mapping EDID fields to Receiver Capabilities in the case Receiver is associated with an Output connected to a downstream counterpart which provides EDID.
 
-_See also the [NMOS Glossary](https://specs.amwa.tv/nmos/branches/main/docs/4.0._Glossary.html), and definitions within RFCs._
+The proposed mapping is affected when the Receiver is:
 
-### Gateway Device
+- Capable of altering signal (e.g. it is able to consume 4K and downscale it before passing to the Output which downstream counterpart supports up to HD resolutions)
+- Constrained beyond the capabilities of the downstream counterpart of the Output (e.g. it supports resolution up to 1080p but the Output is connected to a 4K monitor)
 
-A device which encapsulates a digital video or/and audio signal (usually, from a physical connection) into IP packets. Conversely, the device MAY perform the reverse operation for IP packets to a digital signal.
+The Receiver SHALL include any types of conversion, omit any restrictions, or reflect the internal processing in own Receiver Capabilities.
```

Comment:
Could you explain what this means.

Reply:
It is "SHALL" for circumstances described in the previous paragraph.
```
The proposed mapping is affected when the Receiver is:

- Capable of altering signal (e.g. it is able to consume 4K and downscale it before passing to the Output which downstream counterpart supports up to HD resolutions)
- Constrained beyond the capabilities of the downstream counterpart of the Output (e.g. it supports resolution up to 1080p but the Output is connected to a 4K monitor)
```

Maybe it makes sense to replace these two paragraphs with
```
The proposed mapping SHALL be affected when the Receiver is:

- Capable of altering signal (e.g. it is able to consume 4K and downscale it before passing to the Output which downstream counterpart supports up to HD resolutions)
- Constrained beyond the capabilities of the downstream counterpart of the Output (e.g. it supports resolution up to 1080p but the Output is connected to a 4K monitor)
```

Reply:
Why not say it that way "The proposed mapping provides the mechanism for converting EDID information into Receiver Capabilities. The effective Receiver Capabilities MAY differ from those obtained from the proposed mapping when the Receiver is:
- Capable of altering signal (e.g. it is able to consume 4K and downscale it before passing to the Output which downstream counterpart supports up to HD resolutions) 
- Constrained beyond the capabilities of the downstream counterpart of the Output (e.g. it supports resolution up to 1080p but the Output is connected to a 4K monitor)"

Reply:
Thanks, that's great language.